### ISSUE TITLE
Why inference normalize is needed?

### ISSUE THREAD

**Issue by maoxin7676** [jzi040941/PercepNet#46]
In inference period   the features is multipled by 30，but training period don't have this operation.  Could you explain the reason?

**Comment by jzi040941** [comment]
Yes, you are right, I've removed those feature normalization both inference and train in other branch, feel free to remove it
